.gitignore

```diff
@@ -127,3 +127,6 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+
+# IDEs
+.idea/
```

Comment:
It would be preferable for a PR to only contain changes relevant to the PR itself. Something like this could be addressed in a separate PR if necessary.